packages/expo-notifications/android/src/main/java/expo/modules/notifications/notifications/NotificationSerializer.java

```diff
@@ -159,6 +162,16 @@ private static Bundle toBundle(@Nullable NotificationTrigger trigger) {
     } else {
       bundle.putString("type", "unknown");
     }
+    bundle.putString("channelId", getChannelId(trigger));
+
     return bundle;
   }
+
+  @Nullable
+  private static String getChannelId(NotificationTrigger trigger) {
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
```

Comment:
Is this if necessary here? I guess we could just serialize all data we have, just skip using it whenever it is irrelevant (in older Androids).

One argument that comes to my mind is that user might update it's android version in the meantime. Not sure we are prepared for that, but skipping channelId serialization won't help.

Plus, remember. With every if one dog starves in a shelter.

Reply:
the `getNotificationChannel` looks like this:
```java
@RequiresApi(api = Build.VERSION_CODES.O)
default String getNotificationChannel() 
```
It's annotated with `RequiresApi`, so I need to check if I can call it.